Description: Fixup for python3.3 compat
Author: James Page <james.page@ubuntu.com>
Forwarded: https://github.com/robotis/Lesscpy/pull/23

--- a/lesscpy/scripts/compiler.py
+++ b/lesscpy/scripts/compiler.py
@@ -84,8 +84,9 @@ def run():
     """Run compiler
     """
     aparse = argparse.ArgumentParser(description='LessCss Compiler',
-                                     epilog='<< jtm@robot.is @_o >>',
-                                     version=VERSION_STR)
+                                     epilog='<< jtm@robot.is @_o >>')
+    aparse.add_argument('-v', '--version', action='version',
+                        version=VERSION_STR)
     aparse.add_argument('-I', '--include', action="store", type=str,
                         help="Included less-files (comma separated)")
     aparse.add_argument('-V', '--verbose', action="store_true",
